CHANGELOG.md

```diff
@@ -1,5 +1,200 @@
+- Change names of client type values from presence data [\#136](https://github.com/ably/ably-asset-tracking-swift/issues/136)
+- EnhancedLocationUpdateMessage should be sent and received as a single message, not an array. [\#126](https://github.com/ably/ably-asset-tracking-swift/issues/126)
+- Use Ably and channel state when determining the trackable state in the Subscriber and Publisher SDK [\#120](https://github.com/ably/ably-asset-tracking-swift/issues/120)
+- The trackable state should be updated in the Publisher only after the state is different from the previous one. [\#115](https://github.com/ably/ably-asset-tracking-swift/issues/115)
+- Allow to call Publisher track\(\) method multiple times [\#112](https://github.com/ably/ably-asset-tracking-swift/issues/112)
+- Handle stopped state of the Subscriber [\#109](https://github.com/ably/ably-asset-tracking-swift/issues/109)
+
+**Closed issues:**
+
+- Update `ably-cocoa` dependency [\#299](https://github.com/ably/ably-asset-tracking-swift/issues/299)
+- Update `Subscriber` observer method for location change \(enhanced & raw\) from publisher [\#285](https://github.com/ably/ably-asset-tracking-swift/issues/285)
+- Add next location update intervals to the Subscriber API [\#280](https://github.com/ably/ably-asset-tracking-swift/issues/280)
+- Remove Swift version of `Publisher Example App` [\#278](https://github.com/ably/ably-asset-tracking-swift/issues/278)
+- Replace `CLLocation` with AAT Wrapper `Location` in subscriber delegate [\#267](https://github.com/ably/ably-asset-tracking-swift/issues/267)
+- Use the next location update intervals to make the marker movement smoother in the example app [\#263](https://github.com/ably/ably-asset-tracking-swift/issues/263)
+- Display the horizontal accuracy of locations in the subscriber example app [\#257](https://github.com/ably/ably-asset-tracking-swift/issues/257)
+- Check: Zooming does not improve resolution: Resolution-type events does not get sent [\#255](https://github.com/ably/ably-asset-tracking-swift/issues/255)
+- Update metadata format pushed with location update [\#246](https://github.com/ably/ably-asset-tracking-swift/issues/246)
+- Creating of `SwiftUI` example app [\#231](https://github.com/ably/ably-asset-tracking-swift/issues/231)
+- Update CONTRIBUTING.md about version bumping [\#223](https://github.com/ably/ably-asset-tracking-swift/issues/223)
+- Change `Codable` classes to structs [\#222](https://github.com/ably/ably-asset-tracking-swift/issues/222)
+- Remove Objective-C compatibility [\#218](https://github.com/ably/ably-asset-tracking-swift/issues/218)
+- We need to check how Swift-idiomatic the library is [\#217](https://github.com/ably/ably-asset-tracking-swift/issues/217)
+- Remove unused libs [\#211](https://github.com/ably/ably-asset-tracking-swift/issues/211)
+- Move `.xcworkspace` to `Examples` directory [\#203](https://github.com/ably/ably-asset-tracking-swift/issues/203)
```

Comment:
Doesn't represent a change that carries any value for the SDK user / app developer, so I would remove this line.

Reply:
Removed in [e2cbd82](https://github.com/ably/ably-asset-tracking-swift/pull/322/commits/e2cbd82ad81e7780ba1e00d6cf4f5e581768340d)